examples/models/moshi/mimi/test_mimi.py

```diff
@@ -131,6 +145,34 @@ def forward(self, x):
         ep_decode_output = exported_decode.module()(input)
         self.assertTrue(torch.allclose(ep_decode_output, ref_decode_output, atol=1e-6))
 
+        # PT2E Quantization
+        quantizer = XNNPACKQuantizer()
+        # 8 bit by default
```

Comment:
Why not lower bit? Also how did you decide between pt2e and ao?

Reply:
One step for now. Will add 4 bit via 8da4w.